src/api/schema.ts

```diff
@@ -229,6 +229,12 @@ const rootSchema = `
     campaignId: String!
   }
 
+  type OptOutByCampaign {
+    id: String!
+    title: String!
+    count: String!
```

Comment:
Why is count a string here instead of int?

Reply:
Just seeing this @ajohn25, Most data including IDs are passed back as String especially if they're only going to be displayed and not manipulated in any way. So going with `String!` in most scenarios to keep that convention up.